diff -Naur PythonCard-0.8.2.orig/samples/textIndexer/textIndexer.py PythonCard-0.8.2/samples/textIndexer/textIndexer.py
--- PythonCard-0.8.2.orig/samples/textIndexer/textIndexer.py	2013-11-14 10:13:25.570170409 +0400
+++ PythonCard-0.8.2/samples/textIndexer/textIndexer.py	2013-11-14 10:13:43.302171510 +0400
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 """
 __version__ = "$Revision: 1.28 $"
 __date__ = "$Date: 2005/09/18 03:59:22 $"
diff -Naur PythonCard-0.8.2.orig/samples/textRouter/textRouter.py PythonCard-0.8.2/samples/textRouter/textRouter.py
--- PythonCard-0.8.2.orig/samples/textRouter/textRouter.py	2013-11-14 10:13:25.579170410 +0400
+++ PythonCard-0.8.2/samples/textRouter/textRouter.py	2013-11-14 10:13:43.303171511 +0400
@@ -1,4 +1,4 @@
-#!/usr/local/bin/python
+#!/usr/bin/python
 
 """
 __version__ = "$Revision: 1.85 $"
diff -Naur PythonCard-0.8.2.orig/samples/textRouter/wordwrap.py PythonCard-0.8.2/samples/textRouter/wordwrap.py
--- PythonCard-0.8.2.orig/samples/textRouter/wordwrap.py	2013-11-14 10:13:25.579170410 +0400
+++ PythonCard-0.8.2/samples/textRouter/wordwrap.py	2013-11-14 10:13:43.303171511 +0400
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 #
 # word_wrap.py
 #
diff -Naur PythonCard-0.8.2.orig/samples/webgrabber/webchecker.py PythonCard-0.8.2/samples/webgrabber/webchecker.py
--- PythonCard-0.8.2.orig/samples/webgrabber/webchecker.py	2013-11-14 10:13:25.575170409 +0400
+++ PythonCard-0.8.2/samples/webgrabber/webchecker.py	2013-11-14 10:13:43.303171511 +0400
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#!/usr/bin/python
 
 # Original code by Guido van Rossum; extensive changes by Sam Bayer,
 # including code to check URL fragments.
diff -Naur PythonCard-0.8.2.orig/samples/webgrabber/websucker.py PythonCard-0.8.2/samples/webgrabber/websucker.py
--- PythonCard-0.8.2.orig/samples/webgrabber/websucker.py	2013-11-14 10:13:25.575170409 +0400
+++ PythonCard-0.8.2/samples/webgrabber/websucker.py	2013-11-14 10:13:43.303171511 +0400
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#!/usr/bin/python
 
 """A variant on webchecker that creates a mirror copy of a remote site."""
 
diff -Naur PythonCard-0.8.2.orig/setup.py PythonCard-0.8.2/setup.py
--- PythonCard-0.8.2.orig/setup.py	2013-11-14 10:13:25.557170408 +0400
+++ PythonCard-0.8.2/setup.py	2013-11-14 10:13:43.304171511 +0400
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 # $Id: setup.py,v 1.22 2004/10/03 18:53:22 kasplat Exp $
 # By R.Suzi rnd@onego.ru
 # Extended and expanded by Andy Todd <andy47@halfcooked.com>
diff -Naur PythonCard-0.8.2.orig/tools/codeEditor/codeEditor.py PythonCard-0.8.2/tools/codeEditor/codeEditor.py
--- PythonCard-0.8.2.orig/tools/codeEditor/codeEditor.py	2013-11-14 10:13:25.561170408 +0400
+++ PythonCard-0.8.2/tools/codeEditor/codeEditor.py	2013-11-14 10:13:43.304171511 +0400
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 """
 __version__ = "$Revision: 1.126 $"
 __date__ = "$Date: 2005/12/19 23:18:04 $"
diff -Naur PythonCard-0.8.2.orig/tools/codeEditor/codeEditorR.py PythonCard-0.8.2/tools/codeEditor/codeEditorR.py
--- PythonCard-0.8.2.orig/tools/codeEditor/codeEditorR.py	2013-11-14 10:13:25.561170408 +0400
+++ PythonCard-0.8.2/tools/codeEditor/codeEditorR.py	2013-11-14 10:13:43.304171511 +0400
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 """
 __version__ = "$Revision: 1.14 $"
 __date__ = "$Date: 2004/09/16 21:11:33 $"
diff -Naur PythonCard-0.8.2.orig/tools/codeEditor/restEditor.py PythonCard-0.8.2/tools/codeEditor/restEditor.py
--- PythonCard-0.8.2.orig/tools/codeEditor/restEditor.py	2013-11-14 10:13:25.561170408 +0400
+++ PythonCard-0.8.2/tools/codeEditor/restEditor.py	2013-11-14 10:13:43.304171511 +0400
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 """
 __version__ = "$Revision: 1.7 $"
 __date__ = "$Date: 2004/08/27 02:03:25 $"
diff -Naur PythonCard-0.8.2.orig/tools/oneEditor/codePage.py PythonCard-0.8.2/tools/oneEditor/codePage.py
--- PythonCard-0.8.2.orig/tools/oneEditor/codePage.py	2013-11-14 10:13:25.559170408 +0400
+++ PythonCard-0.8.2/tools/oneEditor/codePage.py	2013-11-14 10:13:43.304171511 +0400
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 """
 __version__ = "$Revision: 1.5 $"
 __date__ = "$Date: 2005/12/29 02:48:25 $"
diff -Naur PythonCard-0.8.2.orig/tools/oneEditor/restEditor.py PythonCard-0.8.2/tools/oneEditor/restEditor.py
--- PythonCard-0.8.2.orig/tools/oneEditor/restEditor.py	2013-11-14 10:13:25.557170408 +0400
+++ PythonCard-0.8.2/tools/oneEditor/restEditor.py	2013-11-14 10:13:43.305171511 +0400
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 """
 __version__ = "$Revision: 1.1 $"
 __date__ = "$Date: 2004/10/03 23:58:01 $"
diff -Naur PythonCard-0.8.2.orig/tools/oneEditor/tabcodeEditor.py PythonCard-0.8.2/tools/oneEditor/tabcodeEditor.py
--- PythonCard-0.8.2.orig/tools/oneEditor/tabcodeEditor.py	2013-11-14 10:13:25.558170408 +0400
+++ PythonCard-0.8.2/tools/oneEditor/tabcodeEditor.py	2013-11-14 10:13:43.305171511 +0400
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 """
 __version__ = "$Revision: 1.15 $"
 __date__ = "$Date: 2005/12/29 02:48:25 $"
